### PR TITLE
Add 404.html, so every link shows a 500 page.

### PR DESCRIPTION
**Pull request by BoomerScratch**
This adds 404.html, so when someone clicks a link, 500 error shows.

### CODE REVIEW

**Comment by R4356th** [comment]
Thank you, @BoomerScratch! This looks cool! But you could you please remove the link to download.scratch.mit.edu and link something funny or odd instead? We are not affiliated with the Scratch Team or Scratch Foundation.

**Comment by BoomerScratch** [comment]
@R4356th Ok!

**Comment by BoomerScratch** [comment]
Ok, when "Download Scratch" is clicked, a meow sound is played. Demo: https://boomerscratch.github.io/glitchy-scratch.github.io/404

**Comment by R4356th** [comment]
Cool! I am merging this right away! Thank you for helping once again! ❤️

File: 404.html [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+        <div class="box-header">
+          <h1>Looks like we are having issues with our servers!</h1>
+        </div>
+        <h3>Try refreshing in a few minutes</h3>
+        <p>You can still <a onclick='fetch("https://assets.scratch.mit.edu/83c36d806dc92327b9e7049a565c6bff.wav").then(res => res.arrayBuffer()).then(arrbuffer => {
+    let context = new AudioContext() || new webkitAudioContext;
```
Comment (BoomerScratch) [inline 404.html:153]: Oops...
```suggestion
    let context = new AudioContext() || new webkitAudioContext();
```

Comment (BoomerScratch) [reply 404.html:153]: Oh, I think it works like the current way.

Comment (R4356th) [reply 404.html:153]: I never noticed this review before. I just fixed this in bfbe8a0 to be consistent.

**Pull request merged**